src/libYARP_OS/src/Os.cpp

```diff
@@ -142,3 +142,22 @@ int yarp::os::fork()
     pid_t pid = yarp::os::impl::fork();
     return pid;
 }
+
+
+#ifdef __APPLE__
```

Comment:
please use `#if defined(__APPLE__)` for coherence with the other parts of the code